pkg/webhook/resources/virtualmachine/validator.go

```diff
@@ -331,6 +335,11 @@ func (v *vmValidator) checkVolumeReq(newVM *kubevirtv1.VirtualMachine) error {
 			}
 			return werror.NewInternalError(fmt.Sprintf("failed to get PVC %s/%s, err: %s", pvcNS, pvcName, err))
 		}
+
+		if imported {
```

Comment:
A question: 

do we have related import mark on golden pvc as well?

suppose an imported VM later refer to other golden pvc, could it break things? looks like a cross check is required.

also, any other parts to block the clonning of the imported golden pvc?